### DISCUSSION TITLE
Travis CI alternatives

### DISCUSSION THREAD

**Discussion by jagadeesr** [QMCSoftware/QMCSoftware#208] [Ideas]
It seems Travis is no longer free to use.
Automated builds and testing stopped for a while.
We need to look for alternatives, GitHub provides some built-in testing " GitHub Actions " 

https://www.jeffgeerling.com/blog/2020/travis-cis-new-pricing-plan-threw-wrench-my-open-source-works

**Comment by schoi32** [comment]
Dear Jags,

The following guidelines may help us move existing automatic tests over to Github Actions:

Migrating from TravisCI to Github Actions
https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions

Sou-Cheng Choi
Research Associate Professor
Department of Applied Mathematics
Illinois Institute of Technology, USA
